.github/workflows/deploy_documentation.yml

```diff
@@ -2,90 +2,83 @@ name: Deploy Documentation
 
 # Controls when the action will run. Triggers the workflow on push
 on:
-  workflow_dispatch:
-#  push:
-#    branches:
-#    - master
-#    paths:
-#    - 'src/docs/*'
-#    - 'project/jsonschema/sssom.schema.json'
+  push:
+    branches: [ master ]
+
+    # paths:
+    # - 'src/docs/*'
+    # - 'project/jsonschema/sssom.schema.json'
 
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  #synchronize:
-  #  runs-on: ubuntu-latest
-  #  steps:
-  #    - uses: actions/checkout@v2
-  #    - name: Merge master to gh-pages branch
-  #      uses: devmasx/merge-branch@v1.1.0
-  #      with:
-  #        type: now
-  #        target_branch: gh-pages
-  #      env:
-  #        GITHUB_TOKEN: ${{ github.token }}
-
-  build:
+  build-docs:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      #----------------------------------------------
-      #       check-out repo and set-up python
-      #----------------------------------------------
-      - name: Check out repository
-        uses: actions/checkout@v2
-        with:
-          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
-          fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+    #----------------------------------------------
+    #       check-out repo and set-up python
+    #----------------------------------------------
+    - name: Check out repository
+      uses: actions/checkout@master
```

Comment:
is it a good idea, to use `master` on any action? What if master breaks?

Reply:
If incoming code from a PR breaks code in `master`, wouldn't that be caught in the PR itself? (assuming the PR is ahead of the master in terms of commits).